src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>En tom fil är inte tillåten.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Domänen kunde inte hittas.</target>
```

Comment:
I'd go with Datatermgruppen here, and @jongotlin @tobiassjosten judging from the term alone. _Domän_ however I feel is more user friendly so that might be best depending on the context and intended audience (and have a similar user-friendliness to what _host_ have).